src/main/java/org/cascadebot/cascadebot/data/objects/GuildSettingsCore.java

```diff
@@ -70,6 +70,9 @@
     @Setting
     private boolean helpShowAllModules = false;
 
+    @Setting
+    private boolean purgePinnedMessages = false;
```

Comment:
This should be in a separate settings file for the moderation module

Reply:
Negative, @weeryan17 Told me ofterwise.

Reply:
No? I told you how settings worked, and where settings where stored not to put this setting in the guild settings core.

Reply:
Is resolved. Check c230b0f2d7d5798bd4fb6c93ea1c9aa4a971e209

Reply:
or c3fefd6ae9648dd9b6d4ebdb3cad4600a18b0b1b